README.md

```diff
@@ -79,6 +79,39 @@ governance](https://pyodide.org/en/stable/project/governance.html).
 - Stack Overflow: [stackoverflow.com/questions/tagged/pyodide](https://stackoverflow.com/questions/tagged/pyodide)
 - Discord: [Pyodide Discord](https://dsc.gg/pyodide)
 
+## Sponsors
+
+### Institutional Support
```

Comment:
I would say drop the subheadings if we're going the route of moving the main info into about.html:
```suggestion
```